charts/linkerd2-cni/templates/cni-plugin.yaml

```
@@ -176,7 +177,9 @@ data:
         ],
         {{- end }}
         "simulate": false,
-        "use-wait-flag": {{.Values.useWaitFlag}}
+        "use-wait-flag": {{.Values.useWaitFlag}},
+        "iptables-mode": {{.Values.iptablesMode | quote}},
+        "ipv6": {{ternary "false" "true" .Values.disableIPv6 }}
```

```suggestion
        "ipv6": {{.Values.disableIPv6 | ternary "false" "true"}}
```

Is this clearer? I find the existing syntax pretty obtuse

---

Yep, that looks a little better :+1: